code/modules/projectiles/projectile/beams.dm

```diff
@@ -46,25 +46,31 @@ In pvp they also have more lasting damages, such as infections, and pain form bu
 
 /obj/item/projectile/beam/weak/pistol_35
 	damage_types = list(BURN = 12)
+	armor_penetration = 5
 	recoil = 1
 
 /obj/item/projectile/beam/weak/light_rifle_257
-	damage_types = list(BURN = 13)
+	damage_types = list(BURN = 18)
+	armor_penetration = 15
 
 /obj/item/projectile/beam/weak/rifle_75
 	damage_types = list(BURN = 22)
+	armor_penetration = 20
 	recoil = 4
 
 /obj/item/projectile/beam/weak/heavy_rifle_408
-	damage_types = list(BURN = 16)
+	damage_types = list(BURN = 20)
+	armor_penetration = 30
 	recoil = 3
 
 /obj/item/projectile/beam/weak/magnum_40
 	damage_types = list(BURN = 20)
+	armor_penetration = 10
 	recoil = 3
 
 /obj/item/projectile/beam/weak/kurtz_50
-	damage_types = list(BURN = 40)
+	damage_types = list(BURN = 30)
```

Comment:
Huge nerf thats really odd as few guns use this

Reply:
It was doing more damage than the standard ammo type that dealt 35 damage.

It went against the theme that laser bullets would otherwise have with other calibers.